src/main/scala/geotrellis/sdg/PopulationNearRoadsJob.scala

```diff
@@ -65,21 +67,33 @@ class PopulationNearRoadsJob(
     new HashPartitioner(partitions)
   }
 
-  // Each buffered road "overflowed", we need to join it back up, not going to trim it tough
-  val roadMaskRdd: RDD[(SpatialKey, MutableArrayTile)] = {
+  val roadsRdd: RDD[(SpatialKey, Seq[Feature[MultiLineString, RoadTags]])] = {
+    val filter = { t: RoadTags => t.isPossiblyMotorRoad /* && t.isStrictlyAllWeather*/ }
```

Comment:
/* && t.isStrictlyAllWeather*/ is confusing, you're trying to include all things that are possibly roads so you can display included and not included roads. Probably worth a small comment too. My first through was "wait, why not use .isIncluded here?"

Reply:
So the commented part was just left over from the old implementation. Are you suggesting that this should be `t.isPossiblyMotorRoad || roadFilter(t)`? That's what would make sense to me. We want to ensure we get the "included" roads, and we then want to include everything else that might be a road for the visualization.